packages/core/src/definitions/styles.ts

```diff
@@ -172,3 +171,114 @@ export const builtInStyles: Record<
     description: 'To open hyperlink in new Tab or not',
   },
 };
+
+export const optionalBuiltInStyles: Partial<
+  Record<ContainerStyleVariableName, ComponentDefinitionVariable<'Text' | 'Boolean'>>
+> = {
+  cfFontSize: {
+    displayName: 'Font Size',
+    type: 'Text',
+    group: 'style',
+    description: 'The font size of the section',
+    defaultValue: '16px',
+  },
+  cfFontWeight: {
+    displayName: 'Font Weight',
+    type: 'Text',
+    group: 'style',
+    description: 'The font weight of the section',
+    defaultValue: 'normal',
+  },
+  cfLineHeight: {
+    displayName: 'Line Height',
+    type: 'Text',
+    group: 'style',
+    description: 'The line height of the section',
+    defaultValue: '20px',
+  },
+  cfLetterSpacing: {
+    displayName: 'Letter Spacing',
+    type: 'Text',
+    group: 'style',
+    description: 'The letter spacing of the section',
+    defaultValue: '0px',
+  },
+  cfTextColor: {
+    displayName: 'Text Color',
+    type: 'Text',
+    group: 'style',
+    description: 'The text color of the section',
+    defaultValue: 'rgba(0, 0, 0, 1)',
```

Comment:
Maybe consider f36 black token? Could be a travis question

Reply:
The default background color style defined in this file uses `rgba` so I wanted to keep it consistent with that

```ts
  cfBackgroundColor: {
    displayName: 'Background',
    type: 'Text',
    group: 'style',
    description: 'The background color of the section',
    defaultValue: 'rgba(255, 255, 255, 0)',
  },
```

I think the SDK does not include F36 tokens because it would increase the bundle size and the SDK is made to be as lightweight as possible.